### PR TITLE
Feature/keepers 2 1 alpha.1 diff

### PR DESCRIPTION
**Pull request by ferglor**

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber_test.go
Comment (amirylm) [inline core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber_test.go:1]: should not be removed

File: core/services/ocr2/plugins/ocr2keeper/evm21/abi_test.go [hunk 1]
```diff
@@ -178,7 +179,7 @@ func TestPacker_UnpackLogTriggerConfig(t *testing.T) {
 				b, _ := hexutil.Decode("0x000000000000000000000000b1182bd20b0537977e945e3e00000000000000000000000000000000000000000000000000000000000000003d53a39550e04688065827f3bb86584cb007ab9ebca7ebd528e7301c9c31eb5d000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")
 				return b
 			}(),
-			automation21Utils.LogTriggerConfig{},
+			iregistry21.KeeperRegistryBase21LogTriggerConfig{},
```
Comment (amirylm) [inline core/services/ocr2/plugins/ocr2keeper/evm21/abi_test.go:182]: `automation_utils_2_1.LogTriggerConfig`

Comment (amirylm) [reply core/services/ocr2/plugins/ocr2keeper/evm21/abi_test.go:182]: same in lines 170, 161

File: core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber_test.go
Comment (ferglor) [reply core/services/ocr2/plugins/ocr2keeper/evm21/block_subscriber_test.go:1]: So, is it correct that the `block_subscriber.go` was deleted (I think it was moved to head.go?), but we want to keep this test file?

**Pull request closed without merging**